demo/sections/components/Tags.vue

```diff
@@ -71,7 +75,7 @@
         <div>
           <p>Inline</p>
           <div class="border border-default p-2 max-w-full">
-            <p-tag-wrapper :tags="numberArr" justify="left" inline class="border border-default" />
+            <p-tag-wrapper :tags="numberArr" justify="left" inline class="border border-default test-123" />
```

Comment:
Guessing `test-123` is something left for debugging

Reply:
Ah yes nice catch